### DISCUSSION TITLE
Unable to Update or Delete user collection records

### DISCUSSION THREAD

**Discussion by AstroArbaaz** [pocketbase/pocketbase#2112] [Q&A] [answered]
req-body: {
  username: 'users94819',
  name: 'tester one',
  email: 'test@example.com',
  phone: '0123456789',
  id: 'ao8dk078he25oia'
}

{
  apiResponseData: {
    code: 404,
    message: "The requested resource wasn't found.",
    data: {}
  }
}

<img width="701" alt="Screenshot 2023-03-21 at 11 00 40 AM" src="https://user-images.githubusercontent.com/43819882/226525964-7c0b5457-dd36-4f0e-b420-89eb90c68ac2.png">

delete using the same Id : 'ao8dk078he25oia'

<img width="703" alt="Screenshot 2023-03-21 at 11 03 15 AM" src="https://user-images.githubusercontent.com/43819882/226526270-b6405d99-ee6e-47cd-a27b-7f594eb954d0.png">

**Comment by AstroArbaaz** [comment, marked as answer]
Solved: Pass the Authorization Token in the headers

**Reply by jacksonsophat** [reply]
Could you show me how pass the Authorization Token?

const data = {name: 'New Name'};
const record = await pb.collection('users').update(pb.authStore.model.id, data);

**Reply by ganigeorgiev** [reply]
@jacksonsophat You don't have to specify explicitly the `Authorization` header if you are using the JS SDK, since it is added automatically if you are authenticated (aka. if `pb.authStore.token` is not empty and `pb.authStore.isValid` returns true).

If your request results to 404 make sure that the user satisfies the Update API rule of your collection.

You can also start the executable with `--debug` for more detailed console error messages and/or check the logs in the Admin UI > Logs.
